tests/basic/misc_test.go

```diff
@@ -1784,4 +1784,62 @@ var _ = Describe("{KubeClusterRestart}", func() {
 		defer EndTorpedoTest()
 		AfterEachTest(contexts, testrailID, runID)
 	})
-})
\ No newline at end of file
+})
+
+var _ = Describe("{VerifyNoPxRestartDueToPxPodStop}", func() {
+	JustBeforeEach(func() {
+		var testrailID = 87311294
+		StartTorpedoTest("VerifyNoPxRestartDueToPxPodStop", "Verify that px serivce remain up even if px pod got deleted ", nil, 0)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	It("has to setup, validate and teardown apps", func() {
+
+		// Get uptime for px service on each node
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			processPid := make(map[string]string)
+			startCmd := "pidof px"
+			//Capturing PID pf PX before stopping PX pods
+			for _, nnode := range node.GetStorageNodes() {
+				output, _ := Inst().N.RunCommand(nnode, startCmd, node.ConnectionOpts{
+					Timeout:         30 * time.Second,
+					TimeBeforeRetry: 20 * time.Second,
+					Sudo:            true,
+				})
+				processPid[nnode.Id] = output
+			}
+			log.Infof(fmt.Sprintf("Process IDs for px before stopping portworx pod  %s", processPid))
```

Comment:
from the logs it says  map as below 
14:58:14  2024-07-16 09:28:07 +0000:[INFO] [{VerifyNoNodeRestartUponPxPodRestart}] [basic.glob..func123.2.1:#1813] - Process IDs for px before stopping portworx pod  map[2b24db67-ef13-4739-b305-4a47e5e15202:23114
14:58:14   e360d6f9-e42f-4a0a-baec-da49d3df6540:30405
14:58:14   f8fb289c-809a-4616-9e59-1fb28405a05a:17421
14:58:14  ]

how are we comparing ProcessPiD in this case ?

Reply:
in the logs attached , we are comparing 
14:58:14  2024-07-16 09:28:07 +0000:[INFO] [{VerifyNoNodeRestartUponPxPodRestart}] [basic.glob..func123.2.1:#1813] - Process IDs for px before stopping portworx pod  map[2b24db67-ef13-4739-b305-4a47e5e15202:23114

14:58:14   e360d6f9-e42f-4a0a-baec-da49d3df6540:30405

14:58:14   f8fb289c-809a-4616-9e59-1fb28405a05a:17421

14:58:14  ]

with 

14:58:19  2024-07-16 09:28:13 +0000:[INFO] [{VerifyNoNodeRestartUponPxPodRestart}] [basic.glob..func123.2.1:#1831] - Process IDs for px after stopping portworx pod  map[2b24db67-ef13-4739-b305-4a47e5e15202:23114

14:58:19   e360d6f9-e42f-4a0a-baec-da49d3df6540:30405

14:58:19   f8fb289c-809a-4616-9e59-1fb28405a05a:17421

14:58:19  ]

but pod restart did not happen .

Reply:
It did happen, i have checked output  of 'kubectl get pods -l name=portworx' and uptime after delete px was just few seconds

Reply:
from the logs seems like PID is not changed

Reply:
Yes, that is what current behaviour of the product, PID does not change due to restart of PX pod